integration/spark-common-test/src/test/resources/cars.csv

```diff
@@ -0,0 +1,4 @@
+name,age
```

Comment:
can you reuse the current csv file, no need to add new one.

Reply:
ok, done